index.html

```diff
@@ -315,57 +317,57 @@
             day = days[1][exception]["schedule"];
         }
     }
-    var currentSlot;
 
-    // Will not get called if there is no school
-    for (currentSlot = 0; currentSlot < days[0][day][school].length; currentSlot++) {
+    if (!days[0][day].imageProperties) {
+      for (var currentSlot = 0; currentSlot < days[0][day][school].length; currentSlot++) {
```

Comment:
I think this should probably get split out into its own function. Generally, if your code gets indented this far, it's a sign it needs to be in its own function.